test/graphinfo.jl

```diff
@@ -36,8 +36,11 @@ A = sparse([0 0 0 0 0; 0 0 0 0 0; 0 0 0 0 0; 0 1 1 0 0; 1 0 0 1 0])
 
 # check the values from the NamedTuple match the values in the fields of GraphInfo
 vals = AbstractPPL.GraphPPL.getvals(model)
-for (i, field) in enumerate([:value, :eval, :kind])
-    @test eval( :( values(m.g.$field) == vals[$i] ) )
+for (i, v) in enumerate(vals[1])
+    @test values(m.g.value[i])[] == Ref(v)[]
+end
+for (i, field) in enumerate([:eval, :kind])
+    @test eval( :( values(m.g.$field) == vals[$i + 1] ) )
```

Comment:
No need for the heavy artillery here...

```suggestion
    @test values(getproperty(m.g, field)) == vals[i + 1] 
```

If ever this becomes necessary, prefer the macro version:

```julia
@eval(values(m.g.$field) == vals[$i + 1])
```